### PR TITLE
Ana/add coinbase referral link in tutorials

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Talking about increasing our revenue streams, this one is a very simple change and can definitely help

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/ModalTutorial.vue [hunk 1]
```diff
@@ -40,6 +40,16 @@
               >
                 {{ item }}
               </span>
+              <span
+                v-if="index === 0"
+                :key="`affiliate-${index}`"
+                class="affiliate-link"
+              >
+                Want some?
+                <a href="https://coinbase-consumer.sjv.io/31vxX"
```
Comment (faboweb) [inline src/components/common/ModalTutorial.vue:49]: should go into config

File: src/components/common/ModalTutorial.vue [hunk 2]
```diff
@@ -40,6 +40,16 @@
               >
                 {{ item }}
               </span>
+              <span
```
Comment (faboweb) [inline src/components/common/ModalTutorial.vue:43]: you added this for all tutorials. shouldn't it only appear in the "get tokens" tutorial?

**Comment by faboweb** [comment]
Nice endevour

File: src/components/common/ModalTutorial.vue (hunk 1 above)
Comment (Bitcoinera) [reply src/components/common/ModalTutorial.vue:49]: yes, you are so right

File: src/components/common/ModalTutorial.vue (hunk 2 above)
Comment (Bitcoinera) [reply src/components/common/ModalTutorial.vue:43]: that's why I have the `v-if="index === 0"` in place

**Comment by Bitcoinera** [comment]
I'm waiting for @jrmoreau approval or suggestions on my little text

**Comment by jbibla** [comment]
should we just have the button go right to coinbase? should we keep the tutorials in? i don't think we have any data about how they're being used. would be good to include a click event.

File: src/components/common/ModalTutorial.vue (hunk 2 above)
Comment (faboweb) [reply src/components/common/ModalTutorial.vue:43]: noooooo
ModalTutorial is used for all tutorials like governance. The link should only appear in the "get tokens" tutorial....

Comment (Bitcoinera) [reply src/components/common/ModalTutorial.vue:43]: Oh no!

**Comment by Bitcoinera** [comment]
> should we just have the button go right to coinbase? should we keep the tutorials in? i don't think we have any data about how they're being used. would be good to include a click event.

Done. Now every click event is sent to GA identified as "linkToCoinbase"

File: src/components/common/ModalTutorial.vue (hunk 2 above)
Comment (Bitcoinera) [reply src/components/common/ModalTutorial.vue:43]: Fixed ✌️

**Comment by Bitcoinera** [comment]
Let's merge! 💃

File: src/components/common/ModalTutorial.vue [hunk 3]
```diff
@@ -41,7 +41,7 @@
                 {{ item }}
               </span>
               <span
-                v-if="index === 0"
+                v-if="step.title === `How to get tokens`"
```
Comment (faboweb) [inline src/components/common/ModalTutorial.vue:44]: 🤮 
What happens if Jordan changes the title of the get tokens step?

Comment (Bitcoinera) [reply src/components/common/ModalTutorial.vue:44]: 😿

**Comment by Bitcoinera** [comment]
Ok, everything fixed in a good & clean way and ready to be merged (and start making some cash 💵 )

File: src/components/common/ModalTutorial.vue (hunk 2 above)
Comment (faboweb) [reply src/components/common/ModalTutorial.vue:43]: Can you explain what you are trying to do here?

Comment (Bitcoinera) [reply src/components/common/ModalTutorial.vue:43]: Salvation! Implements logic for further affiliate links within the `affiliate` property 💃 

Cannot ask for more in one single PR (that originally just aimed to include a simple link) 😉

**Comment by Bitcoinera** [comment]
Let's merge this!

**Pull request merged**